[2025-03-10T13:37:14.786+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: project2.import_csv manual__2025-03-10T13:37:13.449998+00:00 [queued]>
[2025-03-10T13:37:14.794+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: project2.import_csv manual__2025-03-10T13:37:13.449998+00:00 [queued]>
[2025-03-10T13:37:14.794+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-03-10T13:37:14.795+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2025-03-10T13:37:14.795+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-03-10T13:37:14.807+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): import_csv> on 2025-03-10 13:37:13.449998+00:00
[2025-03-10T13:37:14.812+0000] {standard_task_runner.py:55} INFO - Started process 1225 to run task
[2025-03-10T13:37:14.814+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'project2', 'import_csv', 'manual__2025-03-10T13:37:13.449998+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/project2.py', '--cfg-path', '/tmp/tmpxlvp7yxy']
[2025-03-10T13:37:14.815+0000] {standard_task_runner.py:83} INFO - Job 99: Subtask import_csv
[2025-03-10T13:37:14.882+0000] {task_command.py:388} INFO - Running <TaskInstance: project2.import_csv manual__2025-03-10T13:37:13.449998+00:00 [running]> on host d234808dccbd
[2025-03-10T13:37:14.948+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project2
AIRFLOW_CTX_TASK_ID=import_csv
AIRFLOW_CTX_EXECUTION_DATE=2025-03-10T13:37:13.449998+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-03-10T13:37:13.449998+00:00
[2025-03-10T13:37:14.949+0000] {sql.py:254} INFO - Executing: 
        COPY staging_hr FROM '/home/dg/hr_data1.csv' 
        DELIMITER ',' CSV HEADER;
        
[2025-03-10T13:37:14.957+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-03-10T13:37:15.144+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-03-10T13:37:15.164+0000] {sql.py:375} INFO - Running statement: 
        COPY staging_hr FROM '/home/dg/hr_data1.csv' 
        DELIMITER ',' CSV HEADER;
        , parameters: None
[2025-03-10T13:37:15.167+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 266, in execute
    **extra_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 349, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 380, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedTable: relation "staging_hr" does not exist

[2025-03-10T13:37:15.177+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=project2, task_id=import_csv, execution_date=20250310T133713, start_date=20250310T133714, end_date=20250310T133715
[2025-03-10T13:37:15.187+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 99 for task import_csv (relation "staging_hr" does not exist
; 1225)
[2025-03-10T13:37:15.227+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2025-03-10T13:37:15.244+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
